web-registry/src/utils/number/format.ts

```diff
@@ -0,0 +1,12 @@
+type RoundFloatNumberParamsOptions = {
+  decimals?: number;
+};
+
+export const roundFloatNumber = (
```

Comment:
would you be able to add a few unit tests for this @flagrede? i know it's a simple function so i don't consider it super necessary, but for the purpose of understanding what it's doing, seeing some example inputs and outputs would be helpful for me. and perhaps storing those examples as unit tests is nice, rather than just a doc string or something.

Reply:
Added here: https://github.com/regen-network/regen-web/pull/1272/commits/0d4d7a9e32acc64d5816d0ec2cb9dd36cac14dd0